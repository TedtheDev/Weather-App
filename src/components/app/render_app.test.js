import React from 'react';
import App from './app';
import rednerer from 'react-test-renderer';

describe('<App />', () => {
    test('should render correctly', () => {
        const component = rednerer.create(
            <App />
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});