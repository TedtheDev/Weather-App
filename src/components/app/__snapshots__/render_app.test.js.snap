// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should render correctly 1`] = `
<div
  className="sc-bxivhb llhcYp"
>
  <div>
    Hi from Header
  </div>
</div>
`;
